### ISSUE TITLE
Add ۋ to persian layout (Vav with three noqtaí)

### ISSUE THREAD

**Issue by maemunman** [florisboard/florisboard#2102]
This letter is not exactly standard, but it is common enough that the _IranNastaliq_ font — which doesn't even support any diacritics at all — supports this letter. It's purpose is simple: it is the only way to distinguish the V sound from the W sound in writing. There is no diacritic that can make the distinction in its place.

So I request that ۋ be added as a pop-up alternate of و. Thank you for creating this terrific keyboard!

**Comment by Aman9das** [comment]
You should be able top easily contribute to the keyboard layouts.. 

https://github.com/florisboard/florisboard/blob/master/CONTRIBUTING.md#adding-a-new-keyboard-layout

**Comment by anaskaejdar** [comment]
@Aman9das Of course, yes, I can do that. But I am out of town on a long work-related trip at the moment, with only a phone and a barely-functioning tablet. Is there a way I can test custom JSON without being able to run an android emulator?

**Comment by Aman9das** [comment]
https://github.com/florisboard/florisboard/pull/1254

Changing the json is fairly simple, and even if you make mistakes changing it is fairly simple.. for example:

https://github.com/florisboard/florisboard/pull/1254

Back then I also didn't have access to computer to create a test build.. it's alright as long as changes aren't huge :)

**Comment by BardiaB8** [comment]
I don't agree with this. We don't use this letter in Persian, I've actually never seen it before. Unless you need it for special purposes?

**Comment by anaskaejdar** [comment]
> I don't agree with this. We don't use this letter in Persian, I've actually never seen it before. Unless you need it for special purposes?

It is rare, and it is more of a turkic letter, but I have also seen it used in persian calligraphy. It is related to ڤ also (ف with three noqtaí, [wiki](https://wikiless.org/wiki/Ve_(Arabic_letter)?lang=en)), an archaic originally-persian letter, which is still used in kurdish and arabic.
